src/scripts.js

```diff
@@ -348,75 +348,72 @@ $('.show__sleep--btn').on('click', function() {
 });
 
 // Event Listeners for submit post-data buttons
-// $('.sleep__submit--btn').on('click', postSleep());
-// $('.activity__submit--btn').on('click', postActivity());
-// $('.hydration__submit--btn').on('click', postHydration());
-//
-// function postSleep() {
-//   const sleepBody =
-//     {
-//       userID: uniqueUserID,
-//       date: '2023/01/01',
-//       hoursSlept: $('.hours-slept').val(),
-//       sleepQuality: $('.sleep-quality').val()
-//     };
-//
-//   if ($('.hours-slept').length > 0 && $('.sleep-quality').length > 0) {
-//     fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData', {
-//       method: 'POST',
-//       headers: {
-//         'Content-Type': 'application/json'
-//       },
-//       body: JSON.stringify(sleepBody)
-//     })
-//       .then(response => response.json())
-      // .then(data => console.log(data))
-      // .catch(error => console.log('error'))
-//     }
-// };
-
-// function postActivity() {
-//   const activityBody =
-//     {
-//       "userID": uniqueUserID,
-//       "date": today,
-//       "numSteps": $('.num-steps').val(),
-//       "minutesActive": $('.minutes-active').val(),
-//       "flightsOfStairs": $('.flights-stairs').val(),
-//     };
-//
-//   if ($('.num-steps').length > 0 && $('.minutes-active').length > 0 && $('.flights-stairs').length > 0) {
-//     fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData", {
-//       method: 'post',
-//       headers: {
-//           'Content-Type': 'application/json'
-//         },
-//       body: JSON.stringify(activityBody)
-//     })
-//       .then(response => response.json())
-//       .then(data => console.log(data))
-//       .catch(error => console.log('error'))
-//   }
-// };
-//
-// function postHydration() {
-//   const hydrationBody =
-//     {
-//       "userID": uniqueUserID,
-//       "date": today,
-//       "numOunces": $('.num-ounces').val(),
-//     };
-//
-//   if ($('.num-ounces').length > 0) {
-//     fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData", {
-//       method: 'post',
-//       headers: {
-//           'Content-Type': 'application/json'
-//         },
-//       body: JSON.stringify(hydrationBody)
-//     })
-//       .then(response => response.json())
-//       .then(data => console.log(data))
-//       .catch(error => console.log('error'))
-//   }
-// }
+$('.sleep__submit--btn').on('click', postSleep);
```

Comment:
This looks like a lot of edits but it was just uncommenting out the existing code so we could test it bit-by-bit and making the minor changes we worked through together.

-Capitalizing 'POST'
-Changing single quotes to double quotes and vice versa
-Adding event.preventDefault and event arg  **Good looking out @Dunn-Austen!**
-Removing quotes from around the object keys